### DISCUSSION TITLE
How does Tari Universe work under the hood?

### DISCUSSION THREAD

**Discussion by stringhandler** [tari-project/universe#241] [Q&A] [answered]
What is it doing and how is it different from running Tari Launchpad or running the Tari suite manually?

**Comment by stringhandler** [comment, marked as answer]
Tari Universe aims to remove all the set up and configuration involved in mining Tari. 

The basic flow is this:
* TU (Tari Universe) starts up
* TU checks for updates
* TU checks for latest versions of:
  * xmrig  - The most common and efficient Randomx miner
  * xtrgpuminer - Currently the most efficient GPU based sha3x miner
  * minotari_node - Tari base layer node
  * minotari_mmproxy - Tari base layer application that connects xmrig to minotari_node
  * minotari_console_wallet - Tari's base layer wallet
  * shap2pool - A p2pool implementation for Tari
* TU starts up the node, mmproxy and wallet and connects them to each other.
* TU waits for the node to synchronize with the network
* TU optionally starts p2pool depending on settings (Still in development at the time of writing)
* TU starts up xmrig and connects it to the local instance. At this point TU is mining both Monero and Tari
* TU starts up xtrgpuminer and mines Tari on the GPU

**Reply by Mecanik** [reply]
Thanks for the explanation. However, how do you mine solo? Even if I turn off P2P Pool it still mines in a pool. Also, why does it do merge mining? why can't you just mine tari?

And finally, the most important question - what's the difference between wallets create with universe and suite? For example, I downloaded suite and created a wallet. This wallet is not accepted anywhere, not even in universe when I try to send XTM.
